### PR TITLE
Try parallel compose building

### PR DESCRIPTION
**Pull request by jgrund**
Signed-off-by: Joe Grund <jgrund@whamcloud.io>

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -10,13 +10,14 @@ jobs:
...
-        - curl -L https://github.com/docker/compose/releases/download/1.22.0/docker-compose-Linux-x86_64 > docker-compose
+        - curl -L https://github.com/docker/compose/releases/download/1.25.0/docker-compose-Linux-x86_64 > docker-compose
         - chmod +x docker-compose
         - sudo mv docker-compose /usr/local/bin
         - cd docker
         - docker build --rm -t imlteam/manager-service-base:5.1 -f base.dockerfile ../
-        - docker build --rm -t imlteam/manager-nginx -f nginx.dockerfile ../
-        - docker-compose build
+        - docker build --rm -t imlteam/manager-nginx:5.1 -f nginx.dockerfile ../
+        - docker build --rm -t rust-iml-base -f rust-base.dockerfile ../
```
Comment (johnsonw) [inline .travis.yml]: Should this be `imlteam/rust-iml-base`?

File: .travis.yml [hunk 2]
```diff
@@ -87,8 +88,9 @@ jobs:
         - docker build --rm -t imlteam/iml-wasm-components:5.1 -f iml-wasm-components.dockerfile ../
         - docker push imlteam/iml-wasm-components:5.1
         - docker build --rm -t imlteam/manager-nginx:5.1 -f nginx.dockerfile ../
+        - docker build --rm -t rust-iml-base -f rust-base.dockerfile ../
```
Comment (johnsonw) [inline .travis.yml]: `imlteam/rust-iml-base`?

File: .travis.yml (hunk 1 above)
Comment (jgrund) [reply .travis.yml]: I purposely left that off because it's not published. I don't think we should be publishing this (yet).

Comment (johnsonw) [reply .travis.yml]: ok

**Pull request merged**